### PR TITLE
Cgi/transcoding/resource per subtask

### PR DESCRIPTION
**Pull request by weaselix**

### CODE REVIEW

File: golem/task/rpc.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import logging
 import os.path
 import re
+import threading
```
Comment (nieznanysprawiciel) [inline golem/task/rpc.py]: Why is this necessary?

File: golem/task/tasksession.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import enum
 import functools
 import logging
+import threading
```
Comment (nieznanysprawiciel) [inline golem/task/tasksession.py]: Why is this necessary?

File: golem/task/rpc.py (hunk 1 above)
Comment (weaselix) [reply golem/task/rpc.py]: Thanks that you notice that. It is not necessary. It was added for test purposes by me. It was removed.

File: golem/task/tasksession.py (hunk 2 above)
Comment (weaselix) [reply golem/task/tasksession.py]: Same as above.

**Review by nieznanysprawiciel — APPROVED** [review]
